### PR TITLE
Login button for prompt in menubar

### PR DESCRIPTION
**Pull request by gabriel**
https://keybase.atlassian.net/browse/DESKTOP-442

@keybase/react-hackers 

This should be pulled in when login is ready but not before.

### CODE REVIEW

**Comment by chrisnojima** [comment]
we need this to be behind a featureflag

**Comment by chrisnojima** [comment]
i'm adding one for the new login flow, PR soon

File: react-native/react/menubar/index.render.desktop.js [hunk 1]
```diff
@@ -56,19 +56,14 @@ const Footer = props => {
...
-      <Text type='Body' small style={{alignSelf: 'center', marginTop: 6}}>You're logged out!</Text>
-      <Text type='Body' small style={{marginTop: 23, marginBottom: 5, marginLeft: 10}}>From the terminal:</Text>
-      <Terminal>
-        <Text type='TerminalCommand'>keybase login</Text>
-        <Text type='TerminalEmpty'/>
-        <Text type='TerminalComment'>or if you're new to Keybase:</Text>
-        <Text type='TerminalCommand'>keybase signup</Text>
-      </Terminal>
-      {allowLoggedOutKBFS && <Text type='Body' small style={{marginTop: 22, marginBottom: 7, marginLeft: 10}}>Or access someone's public folder:</Text>}
+        <i style={{alignSelf: 'center', color: globalColors.lowRiskWarning, marginTop: 12}} className='fa fa-exclamation-triangle'/>
```
Comment (chrisnojima) [inline react-native/react/menubar/index.render.desktop.js]: we should use the icon class

**Comment by gabriel** [comment]
ok, I think ready

File: shared/menubar/index.js [hunk 2]
```diff
@@ -159,6 +164,7 @@ class Menubar extends Component {
       showMain={() => this.showMain()}
       showHelp={() => this.showHelp()}
       showUser={user => this.showUser(user)}
+      logIn={() => this.logIn()}
```
Comment (chrisnojima) [inline shared/menubar/index.js]: just call showMain() instead of logIn()

Comment (gabriel) [inline shared/menubar/index.js]: There is a todo to show the login screen on main view if some other view is present in the main view... like signup?

**Pull request merged**